python/tvm/relay/op/contrib/clml.py

```diff
@@ -200,9 +218,18 @@ def check_conv(extract):
 
         while call.op.name != "nn.conv2d":
             call = call.args[0]
+
         attrs, args = call.attrs, call.args
         if attrs.data_layout != "NCHW":
             return False
+
+        if (
+            (not isinstance(args[0], (Var, Constant)))
+            and (args[0].op.name == "nn.pad")
+            and (len(args[0].attrs["pad_width"]) != 4)
```

Comment:
Why there is such limitation on padding width?

Reply:
Ideally, this check is not required. nn.pad implementation ensures pad_width length to be in sync with incoming tensor rank.

Reply:
CLML module is expected to have input to 4D tensor shape for Pad layer, since conv2d layer input is always 4D tensor so Pad+conv2d fusion input is expected have 4D tensor always. It seems condition is redundant. Removed in updated commit.
@srkreddy1238 @echuraev